tests/test_main.cpp

```diff
@@ -389,6 +416,8 @@ int main( int argc, const char *argv[] )
 
     if( world_generator == nullptr ) {
         // The session run may not have initialized game state because asked to list tests
+        std::chrono::duration<double> elapsed_seconds = end - start;
+        DebugLog( D_INFO, DC_ALL ) << "Finished in " << elapsed_seconds.count() << " seconds";
```

Comment:
Ideally it should test whether any test is run and return if not, but for the time being this branch should probably be removed for it to print the random seed and errors.